src/lib/rsync/tests/test_rsync_download.py

```
@@ -0,0 +1,126 @@
+from typing import List
+from unittest import mock
+
+from src.lib.rsync import rsync
+
+
+def _make_request(remote_path: str) -> rsync.RsyncRequest:
+    return rsync.RsyncRequest(
+        workflow_id='wf-1',
+        task_name='task-main',
+        direction=rsync.RsyncDirection.DOWNLOAD,
+        local_path='/tmp/dst',
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/dst"

(S108)